db.js

```diff
@@ -0,0 +1,67 @@
+var MongoClient = require('mongodb').MongoClient;
+var express = require('express');
+// var assert = require('assert');
+var router = express.Router();
+
+// An object to hold some parameters of the current db
+var dbcurrent = {
+  url: "mongodb://localhost:27017/",
+  dbname : "test",
+  collection:"users",
+  data:{
+    name:'Ilya'
+  },
+
+  setURL: function (url) {
+    this.url = url;
+    return this.url;
+  },
+
+  setDB : function (db) {
+    this.dbname = db;
+    return this.dbname;
+  }
+}
+
+router.get('/connect', function(req, res){
```

Comment:
This is classic example of `single responsibility principal` violation. NEVER EVER mix two absolute different parts together 

DB interaction has absolutely nothing in common with routing